docker/nanotube/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.12
```

Comment:
This can be bumped to 1.13. I will do appropriate change in Nanotube module setting shortly.